cmd/config.go

```diff
@@ -31,4 +31,5 @@ type Config struct {
 	GrpcClientCertFile        string        `required:"false" desc:"the gRPC client certificate file" envconfig:"grpc_client_cert_file"`
 	GrpcClientKeyFile         string        `required:"false" desc:"the gRPC client key file" envconfig:"grpc_client_key_file"`
 	PXEVlanID                 uint16        `required:"false" default:"4000" desc:"the id of the pxe vlan" envconfig:"pxe_vlan_id"`
+	PodCIDRs                  []string      `required:"false" default:"10.240.0.0/12" desc:"the pod cidrs, one cidr for ipv4 and on for ipv6 separated by comma" envconfig:"pod_cidrs"`
```

Comment:
```suggestion
	PodCIDRs                  []string      `required:"false" default:"10.240.0.0/12" desc:"the pod CIDRs, one CIDR for IPv4 and one for IPv6 separated by comma" envconfig:"pod_cidrs"`
```

Reply:
One possibility would be to add the podcidr as a label to the tenant network during network allocation in the gepm, and pick it here if this annotation is present.

Reply:
Maybe rename to something non kubernetes thing, like DefaultRouteMapCIDRs

Reply:
there can even be more or less than one cidr per AF

Reply:
renaming done, evaluation of network label comes next

Reply:
This is now done on the metal-api side where a property `additionalannouncablecidrs` can be added to the tenant super network, if present, these cidrs are added to the BGPFilter which is then part of the route maps per port per machine